business/apps.go

```diff
@@ -23,6 +24,14 @@ type AppService struct {
 	businessLayer *Layer
 }
 
+type AppCriteria struct {
```

Comment:
Commented in the graph appender work: https://github.com/kiali/kiali/pull/4755/files#diff-942c6d3c3bcb3e799d538be7c6b9dcaab40146ba0435348cf159a5c92b497d2aR369 I think the AppCriteria should include an "IncludeHealth" flag as there may be cases where this method should not return health info (i.e. graph).

Reply:
If I am not wrong, the `Health` field is what is being used for that purpose.

Reply:
Hi, sorry but I am sick and I didn't answer this. Yes the health field is for that.